src/main/java/ch/epfl/biop/wrappers/cellpose/CellposeTaskSettings.java

```diff
@@ -14,20 +14,19 @@
     int diameter = 30;
     double flow_threshold = 0.4;
     double cellprob_threshold = 0.0;
```

Comment:
Do you want the default to be 0 ?

Reply:
from [cellpose documentation ](https://cellpose.readthedocs.io/en/latest/command.html?highlight=cellprob_threshold#options)